client/src/components/geneExpression/geneSet.tsx

```diff
@@ -35,16 +37,29 @@ class GeneSet extends React.Component<{}, State> {
     this.setState({ isOpen: !isOpen });
   };
 
+  onGeneExpressionComplete = (): void => {
+    const { genesetLoadCount } = this.state;
+    // @ts-expect-error ts-migrate(2339) FIXME: Property 'setGenes' does not exist on type 'Readonl... Remove this comment to see the full error message
+    const { setGenes } = this.props;
+    this.setState({ genesetLoadCount: genesetLoadCount + 1 });
+    if (genesetLoadCount + 1 === setGenes.size) {
+      this.setState({ isGeneExpressionComplete: true });
```

Comment:
We probably don't have to use state to capture the isComplete status, just set a variable to true in `componentDidUpdate`